src/TwigComponent/src/Command/TwigComponentDebugCommand.php

```diff
@@ -356,4 +359,17 @@ private function getAnonymousComponentProperties(ComponentMetadata $metadata): a
 
         return $properties;
     }
+
+    private function displayDetailsAboutConfiguration(SymfonyStyle $io): void
+    {
+        $io->section('Configuration of TwigComponent');
+        $io->table(
+            ['Configuration', 'Current value'],
+            [
+                ['anonymous_template_directory', $this->anonymousDirectory],
+                ['short_syntax', $this->withShortSyntax ? 'enabled' : 'disabled'],
+                ['profiler',  $this->withProfiler ? 'enabled' : 'disabled'],
+            ]
+        );
+    }
```

Comment:
Not 100% sure this will bring more value than the `debug:config` command (a bit more complete / with examples, etc...)